dat/spob/lua/proteron_mil_protera_nova.lua

```diff
@@ -0,0 +1,138 @@
+
+function init( spb )
+   return luaspob.init( spb, {
+      std_bribe = 100,
+      msg_granted = {
+         _([["You may land on the Directrix."]]),
+      },
+      msg_notyet = {
+         _([["You may not approach the Directrix."]]),
+      }
+   } )
+end
+
+load = luaspob.load
+unload = luaspob.unload
+
+function can_land()
+   if not var.peek(contact_var) then
+      return false, mem.msg_notyet
+   else
+      return true, mem.msg_granted
+   end
+end
+
+function comm()
+   vn.clear()
+   vn.scene()
+   if var.peek(contact_var) then
+   if faction.playerStanding("Proteron")<0 then
+      local _n, str = faction.get("Proteron"):playerStanding(0)
+      ccomm.newCharacterSpob( vn, mem.spob, {faction_str=str, name_colour=colour.new("Yellow")} )
+   else
+      ccomm.newCharacterSpob( vn, mem.spob)
+   end
+   vn.transition()
+   vn.na(fmt.f(_("You establish a communication channel with the ensigns at {spb}, who chatter excitedly at you."),
+      {spb=mem.spob}))
+
+   vn.label("menu")
+   vn.menu( function ()
+      return {
+         { _("Close"), "leave" }
+      }
+   end )
+   vn.label("leave")
+   vn.run()
+
+   mem.spob:canLand() -- forcess a refresh of condition
+   else
+   ccomm.newCharacterSpob( vn, mem.spob )
+   vn.transition()
+   vn.na(fmt.f(_("You establish a communication channel with {spb}. Some opera seems to be playing in the background with someone shouting at someone else to turn it off. \nYou wonder what base would be this poorly managed."),
+      {spb=mem.spob}))
+
+   vn.menu( function ()
+      return {
+         { _("Close"), "leave" },
+         { _("Bribe"), "bribe" }
+      }
+   end )
+
+   vn.label("bribe")
+   vn.na(_([["How dare you try to bribe the Sovereign Proteron Autarchy!? We would certainly never take a bribe! I have never taken one and I don't know who would either! I -"]]))
+   vn.na(_([["bob, have you started pretending to uphold the ideals of the Proteron to imaginary enemies again?"]])) --bob is intentionally smallcase, do not attempt to change.
+   vn.na(_([["Wait, why isn't the video on? Turn the video on! I want to see which holo you spawned this time!"]]))
+
+   --vn.disappear(spb) --It's no longer a comm channel, it's now a regular conversation
+   vn.scene()
+   vn.func( function() ccomm.cleanup(vn) end)
+   vn.transition()
+   local bob = vn.newCharacter(_("bobblehat"), {image="neutral/male2n.webp", color=nil, shader=love_shaders.hologram(), pos="left"})
+   local Jen = vn.newCharacter(_("Jennet"), {image="neutral/female1n.webp", color=nil, shader=love_shaders.hologram(), pos="right"})
+   Jen(_([[Oh. This isn't a holo, is it? This is real. You weren't playing around this time.]])) --Double quotes dropped to highlight the context change (usual comm -> convo)
+   bob(_([[For your information, ensign Jennet, I was never "playing around". I was practicing in order to better defend our glorious Autarchy.]]))
+   Jen(_([[Riight, of course... "Practising", huh? I'll keep that in mind.]]))
```

Comment:
Codespell won't like "Riight". But it definitely is correct in the context, so
```suggestion
   Jen(_([[Riight, of course... "Practising", huh? I'll keep that in mind.]])) -- codespell:ignore Riight
```
It may also dislike "Practising"

Reply:
I'll wait on practising, but I'll add the ignore.